default_config.yaml

```diff
@@ -0,0 +1,53 @@
+includes: ["GitHub", "GitHubCommunityStandards", "GitHubRulesets"]
```

Comment:
I'm torn on this change. On one hand, I'd like to see a default configuration file where all of the options match the default values; in this case, I'd like to see it only include line #1. On the other hand, this config file does a good job illustrating the type of information that can be included in a config file. 

So, my recommendation is to reduce this config file to line #1 to support the first scenario, and then create an issue that introduces functionality to generate this config file when `RepoAuditor` is invoked with a special command or flag. We can then prioritize this issue appropriately in the future.

What are your thoughts on this?

Reply:
Sounds good to me, kind of like how a tool like [poetry](https://python-poetry.org/) asks for project information before generating all relevant files.

We can debate the complexity of this approach in the corresponding issue. 🙂

Reply:
As discussed in today's team meeting, the issues we face are:
1. The information in the config file is redundant since the same info is provided by `repo_auditor --help`.
2. Third party plugins can exist which would need to have their flags in the default config. We have no way of knowing this, and we should look at how `pytest` handles this.

An alternate solution is to create a simple webpage that details all the config options provided by us along with example usage in both CLI and the config. @davidbrownell seems to like this idea the most.

Reply:
I updated the default config to have the minimum set of values needed to run, which involves `includes`, `url`s, and the `GitHub-pat`.